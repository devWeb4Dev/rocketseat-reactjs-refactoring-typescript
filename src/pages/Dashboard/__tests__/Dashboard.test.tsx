import renderer from 'react-test-renderer'

import { Dashboard } from '../index'

describe('Counter', () => {
  test('Link changes the class when hovered', () => {
    const component = renderer.create(<Dashboard />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
