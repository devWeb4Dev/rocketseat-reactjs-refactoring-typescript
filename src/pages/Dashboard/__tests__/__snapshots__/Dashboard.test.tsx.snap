// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Link changes the class when hovered 1`] = `
Array [
  <div
    className="sc-gsDKAQ hsciCP"
  >
    <header>
      <img
        alt="GoRestaurant"
        src="logo.svg"
      />
      <nav>
        <div>
          <button
            onClick={[Function]}
            type="button"
          >
            <div
              className="text"
            >
              Novo Prato
            </div>
            <div
              className="icon"
            >
              <svg
                fill="none"
                height={24}
                stroke="currentColor"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
                style={
                  Object {
                    "color": undefined,
                  }
                }
                viewBox="0 0 24 24"
                width={24}
                xmlns="http://www.w3.org/2000/svg"
              >
                <rect
                  height="18"
                  rx="2"
                  ry="2"
                  width="18"
                  x="3"
                  y="3"
                />
                <line
                  x1="12"
                  x2="12"
                  y1="8"
                  y2="16"
                />
                <line
                  x1="8"
                  x2="16"
                  y1="12"
                  y2="12"
                />
              </svg>
            </div>
          </button>
        </div>
      </nav>
    </header>
  </div>,
  <div
    className="sc-jRQBWg dQUtxZ"
    data-testid="foods-list"
  />,
]
`;
